### PR TITLE
CIV-9156 Claim intent claimant notification

### PR DESCRIPTION
**Pull request by hmcts-version1-domini**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-9156

### Change description ###

Claim intention applicant notification
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseCuiTest.java [hunk 1]
```diff
@@ -23,6 +23,12 @@ public class ClaimantResponseCuiTest extends BpmnBaseTest {
     //Activity IDs
     private static final String NOTIFY_LIP_RESPONDENT_CLAIMANT_CONFIRM_TO_PROCEED_ACTIVITY_ID
         = "NotifyLiPRespondentClaimantConfirmToProceed";
+    private static final String NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED
+        = "NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED";
```
Comment (neeta-hmcts) [inline src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseCuiTest.java:27]: move this to line no 22

File: src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseCuiTest.java [hunk 2]
```diff
@@ -23,6 +23,12 @@ public class ClaimantResponseCuiTest extends BpmnBaseTest {
...
+
+    //Activity IDs
```
Comment (neeta-hmcts) [inline src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseCuiTest.java:29]: move this to line no 28

File: src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseCuiTest.java (hunk 1 above)
Comment (hmcts-version1-domini) [reply src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseCuiTest.java:24]: Done

File: src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseCuiTest.java (hunk 2 above)
Comment (hmcts-version1-domini) [reply src/test/java/uk/gov/hmcts/reform/civil/bpmn/ClaimantResponseCuiTest.java:26]: Done

**Review by neeta-hmcts — APPROVED** [review]

**Review by jeswanth-hmcts — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Review by rajakm — APPROVED** [review]

**Pull request merged**